src/ngAutocomplete.js

```diff
@@ -148,6 +148,12 @@ angular.module( "ngAutocomplete", [])
           }
         }
 
+        if (scope.options.getPlaceOnBlur) {
+          element.bind('blur', function () {
```

Comment:
Could rely on check if the autocomplete selection was not fired? otherwise bypass de blur event